packages/utils/src/englishConjugation/conjugateEnglish.ts

```diff
@@ -31,13 +31,37 @@ function buildVerbParts(root: string): EnglishVerbComponents {
 function buildPast(parts: EnglishVerbComponents): EnglishVerbTense {
   const verbTense = {
     continuous: {
-      I: `was ${parts.presentParticiple}`,
-      you: `were ${parts.presentParticiple}`,
-      it: `was ${parts.presentParticiple}`,
-      we: `were ${parts.presentParticiple}`,
-      youAll: `were ${parts.presentParticiple}`,
-      they: `were ${parts.presentParticiple}`,
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
     },
+    simple: {
+      I: `was ${parts.pastSimple}`,
+      you: `were ${parts.pastSimple}`,
+      it: `was ${parts.pastSimple}`,
+      we: `were ${parts.pastSimple}`,
+      youAll: `were ${parts.pastSimple}`,
+      they: `were ${parts.pastSimple}`,
+    },
+    perfect: {
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
+    },
+    perfectContinuous: {
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
+    }
```

Comment:
awesome job with this.  default initialization was definitely the error and you hit the nail on the head!

The loop below on L#67-L#73 will take care of building the actual values for us.  How do you feel about going with the suggestion below?

```suggestion
    simple: {},
    perfect: {},
    perfectContinuous: {}
```